scripts/policy-paths.yaml

```diff
@@ -0,0 +1,18 @@
+---
+policies:
+  namespace: openshift-rbac-policies
+  paths:
+    - rbac-permissions-operator-config
+    - osd-cluster-admin
+    - backplane
+    - backplane/srep
+    - ccs-dedicated-admins
+    - customer-registry-cas
+    - osd-openshift-operators-redhat
+    - osd-pcap-collector
+    - osd-limited-support
+
+limited-support-policies:
+  namespace: openshift-limited-support-policies
```

Comment:
Hey Ben, this new Namespace needs to be created like this 
 `so if the new policies don't belong to openshift-managed-rbac-config, create a manifest to create a new namespace. Example: https://github.com/openshift/managed-cluster-config/blob/master/deploy/acm-policies/00-openshift-managed-rbac-policies.Namespace.yaml`